src/attributes/prop.js

```diff
@@ -79,6 +79,12 @@ jQuery.extend( {
 	}
 } );
 
+// Support: IE<=11
```

Comment:
What about Edge? I'd love all new IE-specific stuff to be tested on Edge and if the bug is not present there to write `IE <= 11 only`.

Adding spaces around `<=` would be good as well, that's where the conventions [seem to go currently](https://github.com/jquery/contribute.jquery.org/issues/95#issuecomment-69379197).